src/OrchardCore.Modules/OrchardCore.Users.EntityFrameworkCore/Module.txt

```diff
@@ -0,0 +1,10 @@
+Name: Users (Entity Framework Core stores)
+AntiForgery: enabled
+Author: The Orchard Team
+Website: http://orchardproject.net
+Version: 2.0.x
+OrchardVersion: 2.0.x
+Description: The Users Entity Framework Core module allows to store user data via Entity Framework Core ORM
+FeatureDescription: Entity Framework Core Stores for the Users module.
```

Comment:
For infos.

For the module default feature or for a feature defined in the features section, the description key is `Description`. Currently, through the code the `FeatureDescription` key is never used. E.g, in the `Modules` page only `Description` values of features (could be the module default one) are showed up.

So, i'm used to remove the `FeatureDescription` line and just use the `Description` line with the most descriptive text.